homeassistant/components/homekit/accessories.py

```diff
@@ -85,34 +85,57 @@ def set_accessory_info(acc, name, model, manufacturer=MANUFACTURER,
 class HomeAccessory(Accessory):
     """Adapter class for Accessory."""
 
-    # pylint: disable=no-member
+    category = 'OTHER'
 
-    def __init__(self, name=ACCESSORY_NAME, model=ACCESSORY_MODEL,
-                 category='OTHER', **kwargs):
+    def __init__(self, hass, name, entity_id, config, **kwargs):
         """Initialize a Accessory object."""
         super().__init__(name, **kwargs)
```

Comment:
Is this **kwargs or the combination of config and aid that should be passed here?

Reply:
I will change `**kwargs` to `aid`.